docs/admin/guides/rbac.md

```diff
@@ -54,6 +54,15 @@ protect who can download Python content then do so by adding a content guard to
 pulp python distribution update --name foo --content-guard $CONTENT_GUARD_HREF_OR_NAME
 ```
 
+Example pulp-cli workflow to add RBAC-based access to download from the index:
+
+```bash
+pulp content-guard rbac create --name foo-guard
+pulp content-guard rbac assign --name foo-guard --user user1 --user user2 --group group1 --group group2
+CG_HREF=$(pulp content-guard rbac list --name foo-guard | jq -r ".[0].pulp_href")
+pulp python distribution update --name foo --content-guard $CG_HREF
+```
```

Comment:
does pip allow to use authentication when consuming this?
Would it be easy enough to add it to this example?

Reply:
Yep it sure does.  I added links to docs for common package managers on how to use basic auth instead of just a pip example.  That way the "howto" does not have to be maintained here in the pulp_python docs over time.